test/Game-test.js

```diff
@@ -1,19 +1,58 @@
 const chai = require('chai');
 const expect = chai.expect;
-const Player = require('../Game.js');
+const Game = require('../lib/Game.js');
 const spies = require('chai-spies');
 chai.use(spies);
 
-global.domUpdates = require('../domUpdates.js');
+global.domUpdates = require('../lib/domUpdates.js');
 chai.spy.on(global.domUpdates, [], () => true);
 
-describe ('Game', function() {
+describe('Game', function() {
   var game;
   beforeEach(function() {
-  game = new Game();
-});
+    game = new Game();
+  });
+
+  it('should store the current round, which should start at round 1', function() {
+    expect(game.currentRound).to.equal(1);
+  });
+
+  it('should store the current players turn, which should start at player 1', function() {
+    expect(game.playerTurn).to.equal(1);
+  });
+
+  it('should store an array of category IDs, which should start as an empty array', function() {
+    expect(game.categoryIds).to.deep.equal([]);
+  });
```

Comment:
You can lump all these 'default property' assertions into a single it block rather than separating them out